apps/reader/src/components/viewlets/AnnotationView.tsx

```diff
@@ -51,8 +52,49 @@ const AnnotationPane: React.FC = () => {
     )
   }, [focusedBookTab?.book.annotations])
 
+
+  const exportAnnotations = () => {
+    const annotations = (focusedBookTab?.book.annotations ?? []).map((a) => ({ ...a }))
+    // process annotations to be under each section
+    // group annotations by title
+    const grouped = group(annotations, (a) => a.spine.title)
+    const exported: Record<string, any[]> = {}
+    for (const chapter in grouped) {
+      const annotations = grouped[chapter]?.map((a) => {
+        const annotation: Record<string, any> = {}
+        if (a.notes !== undefined) annotation.notes = a.notes
+        if (a.text !== undefined) annotation.text = a.text
+        return annotation
+      }) ?? []
+      exported[chapter] = annotations
+    }
+
+    // Copy to clipboard as markdown
+    const exportedAnnotationsMd = Object.entries(exported).map(([chapter, annotations]) => {
+      return `## ${chapter}\n${annotations.map((a) => `- ${a.text} ${a.notes ? `(${a.notes})` : ''}`).join('\n')}`
+    }).join('\n\n')
+    navigator.clipboard.writeText(exportedAnnotationsMd)
+
+    // Download as markdown
+    const blob = new Blob([exportedAnnotationsMd], { type: 'text/markdown' })
+    const url = URL.createObjectURL(blob)
+    const a = document.createElement('a')
+    a.href = url
+    a.download = 'exported_annotations.md'
+    a.click()
+  }
+
   return (
-    <Pane headline={t('annotations')}>
+    <Pane headline={t('annotations')} actions={Object.keys(groupedAnnotation).length > 0 ? [
```

Comment:
conditionally render the copy annotation icon

Reply:
<img width="255" alt="image" src="https://github.com/user-attachments/assets/d6761ee5-ccf9-4491-8454-f89d82c0cf35">